### PR TITLE
iPhone SE Wallet Address Label Overflow

### LINKED ISSUE #119: iPhone SE wallet address overflow
Needs to wrap onto 2 lines on SE

### PR DESCRIPTION
**Pull request by borwahs**
- Allow text to shrink to min factor.
- Set leading/trailing edges to line up with qr code.
- Fix in modal and table view cell.

Fixes #119.

### CODE REVIEW

**Comment by borwahs** [comment]
Here is what it looks like in simulator for iPhone SE.

Screen Shot 2019-03-29 at 5 15 41 PM

**Comment by ricburton** [comment]
🤘🤘🤘

**Comment by borwahs** [comment]
I somehow missed the table view cell.

Screen Shot 2019-03-29 at 6 08 50 PM

File: Balance/Views/WalletInfoView.swift [hunk 1]
```diff
@@ -160,6 +163,8 @@ class WalletInfoView: UIView {
             make.top.equalTo(qrCodeImageView.snp.bottom).offset(20)
             make.centerX.equalToSuperview()
             make.bottom.equalToSuperview().offset(-20)
+            make.leading.equalTo(qrCodeImageView.snp.leading)
```
Comment (einsteinx2) [inline Balance/Views/WalletInfoView.swift]: When you're using the same property, you don't need to specify it. So these lines should be:

```
make.leading.equalTo(qrCodeImageView)
make.trailing.equalTo(qrCodeImageView)
```

**Review by einsteinx2 — CHANGES_REQUESTED** [review]

File: Balance/Views/WalletInfoView.swift (hunk 1 above)
Comment (borwahs) [reply Balance/Views/WalletInfoView.swift]: slick! haven't used snapkit before so still getting the API down.

**Comment by ricburton** [comment]
Screen Shot 2019-03-30 at 05 41 18

I wonder if the address would look better on two lines.

**Comment by borwahs** [comment]
Is this the Wallet Table View? It should be shrinking it down to fit in the constraints. I can try seeing what it looks like with 2 lines. I’ll post an image when I get a minute.

**Comment by borwahs** [comment]
@ricburton  //  how does this look when using a monospace font?

Screen Shot 2019-03-30 at 11 45 43 AM

**Comment by ricburton** [comment]
Oh yes mono for days. Home soon!

Richard

**Comment by borwahs** [comment]
I'll open a new PR with the monospace font changes since it this is tied to the overflow issue.

**Review by einsteinx2 — APPROVED** [review]

**Pull request merged**